modules/tests/test/copy_template_test.go

```diff
@@ -238,3 +290,51 @@ var _ = Describe("Copy template task", func() {
 		})
 	})
 })
+
+func checkRemovedRecordsTemplate(obj map[string]string) {
+
+	for record, _ := range obj {
+
+		if strings.HasPrefix(record, templates.TemplateOsLabelPrefix) {
+			Expect(record).To(Equal(""), fmt.Sprintf("there should be no %s labels", templates.TemplateOsLabelPrefix))
+		}
+
+		if strings.HasPrefix(record, templates.TemplateFlavorLabelPrefix) {
+			Expect(record).To(Equal(""), fmt.Sprintf("there should be no %s labels", templates.TemplateFlavorLabelPrefix))
+		}
+
+		if strings.HasPrefix(record, templates.TemplateWorkloadLabelPrefix) {
+			Expect(record).To(Equal(""), fmt.Sprintf("there should be no %s labels", templates.TemplateWorkloadLabelPrefix))
+		}
```

Comment:
You can use `HavePrefix()` matcher instead of the `if` statements. For example:
```go
Expect(record).ToNot(HavePrefix(templates.TemplateOsLabelPrefix), fmt.Sprintf("there should be no %s labels", templates.TemplateOsLabelPrefix))
```